about.html

```diff
@@ -68,7 +68,7 @@ <h2>ABOUT ME</h2>
 
 </p>
                 <br>
-                <a href="" class="homeBtn">HIRE ME</a>
+                <a href="https://mail.google.com/mail/u/0/#sent?compose=GTvVlcSHxTXzSsMSLMZPVlDQZLztRrhSVNCjGGpfLdLNklTNSNzSJVnMPCJWgCgpHwgKgXBVDFJWM" class="homeBtn">HIRE ME</a>
```

Comment:
it's not the right way to do it. @himanshu07gupta. 
href="mailto:kumaridivya2796@gmail.com"
remove href with it.

Reply:
ok sure